src/views/TrackPositionAnimation/TrackPositionAnimationView.tsx

```diff
@@ -3,10 +3,15 @@ import { TwoDTransformProps, use2DTransformationMatrix, useAspectTrimming } from
 import { Margins } from 'FigurlCanvas/Geometry'
 import { matrix, Matrix, multiply, transpose } from 'mathjs'
 import React, { FunctionComponent, useCallback, useEffect, useMemo } from "react"
+import { BOOKMARK_BUTTON } from 'views/common/Animation/AnimationControls/PlaybackBookmarkButton'
+import { CROP_BUTTON } from 'views/common/Animation/AnimationControls/PlaybackCropWindowButton'
+import { PlaybackOptionalButtons } from 'views/common/Animation/AnimationControls/PlaybackOptionalButtons'
+import { SYNC_BUTTON } from 'views/common/Animation/AnimationControls/PlaybackSyncWindowButton'
```

Comment:
Optional configuration is now in the implementing class, rather than being hard-coded in the base `FrameAnimation` class (at which point it isn't actually optional).